src/lib/peer.ts

```diff
@@ -27,10 +28,8 @@ export class PeerAPI extends BaseAPI {
     }
 
     this.cache[address] = this.lambdasClient
-      .fetchProfiles([address.toLowerCase()], {
-        headers: { 'Cache-Control': 'max-age=0' }
-      })
-      .then(profiles => profiles[0])
+      .getAvatarsDetailsByPost({ ids: [address.toLowerCase()] })
```

Comment:
is the cache control no needed anymore?

Reply:
I think that as the request is done by the POST method, the request is not cached by the browser, so no need of adding the cache control header. Besides that, the new function doesn't accept sending headers